### ISSUE TITLE
'sort by' and 'search' features not working

### ISSUE THREAD

**Issue by Fren62** [ttdyce/NHentai-NHViewer#30]
Whenever i try to search something or i sort the manga for popularity the app reload the same frontpage of the latest relased. I'm in version 3.3.2 and tried refreshing cookies many times.

**Comment by ttdyce** [comment]
Hi, I have tested things were ok on my device. 

I guess you have chosen the "All" language. 
If so, the "sort by popularity" function is not supported by the site. 

But for the searching part, it seems to be a bug with "All" language, I might look into the code later on. 

Thanks for reporting!

**Comment by Fren62** [comment]
Thanks. So there will never be a way to sort by popularity all the manga in the site?

**Comment by Fren62** [comment]
Sorry, I'm not used to github

So there will never be a way to sort by popularity all the manga in the site? Thanks for the reply

**Comment by ttdyce** [comment]
Yes. As far as I know, the site don't provide this kind of data. 
Thanks.
